integration_tests/grpc_server/test_docker_grpc_server.py

```diff
@@ -0,0 +1,136 @@
+import logging
+import os
+from typing import Text, List, Tuple
+
+import grpc
+import pytest
+
+from google.protobuf.json_format import MessageToDict
+from grpc_health.v1 import health_pb2
+from grpc_health.v1.health_pb2_grpc import HealthStub
+
+from rasa_sdk.grpc_py import action_webhook_pb2_grpc, action_webhook_pb2
+from rasa_sdk.grpc_server import GRPC_ACTION_SERVER_NAME
+from integration_tests.conftest import ca_cert, client_key, client_cert
+
+GRPC_HOST = os.getenv("GRPC_ACTION_SERVER_HOST", "localhost")
+GRPC_NO_TLS_PORT = os.getenv("GRPC_NO_TLS_PORT", 7010)
+GRPC_TLS_HOST = os.getenv("GRPC_ACTION_SERVER_TLS_HOST", "localhost")
+GRPC_TLS_PORT = os.getenv("GRPC_TLS_PORT", 7011)
+GRPC_REQUEST_TIMEOUT_IN_SECONDS = os.getenv("GRPC_REQUEST_TIMEOUT_IN_SECONDS", 5)
+
+logger = logging.getLogger(__name__)
+
+
+def create_no_tls_channel() -> grpc.Channel:
+    """Create a gRPC channel for the action server."""
+    logger.info(f"Creating insecure channel to {GRPC_HOST}:{GRPC_NO_TLS_PORT}")
+
+    return grpc.insecure_channel(
+        target=f"{GRPC_HOST}:{GRPC_NO_TLS_PORT}",
+        compression=grpc.Compression.Gzip)
```

Comment:
Should the arguments be all in one line? Wondering how this passed the Code Quality 🤔

Reply:
They would go beyond 88 character line limit if all arguments were in one line.

Reply:
Running `ruff format` for this code formats it as

```
    return grpc.insecure_channel(
        target=f"{GRPC_HOST}:{GRPC_NO_TLS_PORT}", compression=grpc.Compression.Gzip
    )
```

with the line length of 84.

Reply:
Nitpick anyways. I know if this was `rasa-private` that the Code Quality would fail for this. Just raising it for the concern if the Code Quality check is proper for `rasa-sdk`.

Reply:
Ah, yes, but we are using `black` as formatter in Rasa SDK ATM.

Reply:
There is a PR to switch to ruff and remove black alltogether.